automation_testing/tests/dataview/filter.spec.ts

```diff
@@ -0,0 +1,228 @@
+import { expect, test } from "@playwright/test";
```

Comment:
Please pull from develop and push again. This file has already been reviewed in another PR.